### ISSUE TITLE
Uncaught FileNotFoundError in __init__.py line 379

### ISSUE THREAD

**Issue by ATOMPEAR** [Aider-AI/aider#2142]
Aider version: 0.60.0
Python version: 3.12.7
Platform: Windows-11-10.0.27729-SP0
Python implementation: CPython
Virtual environment: Yes
OS: Windows 11 (64bit)
Git version: git version 2.47.0.windows.2

An uncaught exception occurred:

```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "__main__.py", line 7, in <module>
    sys.exit(main())
             ^^^^^^
  File "main.py", line 776, in main
    coder.run()
  File "base_coder.py", line 730, in run
    self.run_one(user_message, preproc)
  File "base_coder.py", line 773, in run_one
    list(self.send_message(message))
  File "base_coder.py", line 1223, in send_message
    lint_errors = self.lint_edited(edited)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
  File "base_coder.py", line 1320, in lint_edited
    errors = self.linter.lint(self.abs_root_path(fname))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "linter.py", line 116, in lint
    res += tree_context(rel_fname, code, lintres.lines)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "linter.py", line 232, in tree_context
    context = TreeContext(
              ^^^^^^^^^^^^
  File "grep_ast.py", line 47, in __init__
    parser = get_parser(lang)
             ^^^^^^^^^^^^^^^^
  File "core.pyx", line 19, in tree_sitter_languages.core.get_parser
  File "core.pyx", line 14, in tree_sitter_languages.core.get_language
  File "__init__.py", line 132, in __init__
    self.lib = cdll.LoadLibrary(fspath(path_or_ptr))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "__init__.py", line 460, in LoadLibrary
    return self._dlltype(name)
           ^^^^^^^^^^^^^^^^^^^
  File "__init__.py", line 379, in __init__
    self._handle = _dlopen(self._name, mode)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: Could not find module 'D:\WORKSPACES\01_AIDER-PROJECT\env1\Lib\site-packages\tree_sitter_languages\languages.dll' (or one of its dependencies). Try using the full path with constructor syntax.

```

**Comment by ATOMPEAR** [comment]
I'm not sure what happened. I just started playing around with aider. It crashed and prompted me to submit this issue with no other input asked of me so I thought I would make this comment

**Comment by ATOMPEAR** [comment]
I was messing with rich and click CLI apps

**Comment by paul-gauthier** [comment]
Thanks for trying aider and filing this issue. 

How did you install aider? What environment are you using? It looks like you don't have the correct set of dependencies installed. This doc may be helpful:

https://aider.chat/docs/troubleshooting/imports.html

I'm going to close this issue for now, but feel free to add a comment here and I will re-open. Or feel free to file a new issue any time.

**Comment by ATOMPEAR** [comment]
I was using python 3.12.7. I had python 12.13.0 on my system but aider didn't work with it so I installed 3.12.7 without adding to path as that would mess with the current version I had installed. I made a env with the 3.12.7 to use with aider in the project. I'm pretty new to aider and python programing so this was the first time I've ever made a env and all that, it has not come up in my endeavors of learning python yet until I was messing around with aider. I installed it by fallowing the instructions? I'm not sure what else to say as far as that. Very well could be something I did thank you for the response. I just felt like it was a good idea to submit this in the case that it might help you guys in some way :)
